Resources/Prototypes/GameRules/events.yml

```diff
@@ -241,17 +241,17 @@
     maxDuration: 120
   - type: PowerGridCheckRule
 
-# - type: entity # DeltaV - Removes the random sentience.
```

Comment:
This weight needs to be dropped to 1 or something; sentient vending machines are 0 RP.